src/components/organisms/Form/FormCheckboxField/FormCheckboxField.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+import CheckboxField from '../../../molecules/CheckboxField/CheckboxField';
+import { IField } from '../../../types';
+
+import useFieldContext from '../useFieldContext';
+
+interface IFormCheckboxFieldProps extends Partial<IField> {
+  name: string;
+}
+
+const FormCheckboxField: FC<IFormCheckboxFieldProps> = ({ name, inputProps, ...rest }) => {
+  const { error, touched, value, onChange, onBlur } = useFieldContext(name);
+  const handleChange = e => {
+    onChange(e.target.checked);
+  };
```

Comment:
Seems that [`onChange`](https://github.com/zopaUK/react-components/blob/fa92a3368672923c50a3a022a0ede7f217dd0cfb/src/components/hooks/useForm/useForm.ts#L50) could have this signature instead of `onChange: (value: any) => void;` ?

Otherwise we have to create this wrapper on every `<Form />` component just to not have to do it on the `useForm` hook?  🧐  💅🏻